src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -870,6 +870,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Text\StringBuilder.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Text\StringBuilder.Debug.cs" Condition="'$(Configuration)' == 'Debug'" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Text\StringRuneEnumerator.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Text\TranscodingStream.cs" />
```

Comment:
Can we delete / replace the Transcoding*Stream types in https://github.com/dotnet/runtime/tree/master/src/libraries/System.Net.Http.Json/src/System/Net/Http/Json now?

Reply:
Ah, good catch. When targeting 5.0 they should be able to use the new public API. When targeting downlevel I suppose we can conditionally include this .cs file in their .csproj. Will need to see if any compilation errors fall out as a result.